### ISSUE TITLE
Accessing Nested Objects - get a hint - link private?

### ISSUE THREAD

**Issue by nsuchy** [freeCodeCamp/freeCodeCamp#15609]
#### Challenge Name

Accessing Nested Objects (https://www.freecodecamp.com/challenges/accessing-nested-objects)

#### Issue Description

The get a hint button is linked to a thread that's private.

#### Browser Information

* Browser Name, Version: Safari
* Operating System: macOS
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
Not relevant to the issue


```
#### Screenshot

![Image of issue](http://i.imgur.com/CHBslHD.png)

**Comment by dhcodes** [comment]
I'm going to have Michael update the name of the forum guide page. That's what's causing this issue.

**Comment by nsuchy** [comment]
Thank you

**Comment by m-henderson** [comment]
I work fix this asap. Will update you all when it is taken care of.

**Comment by m-henderson** [comment]
do you have the url to the form that needs to be changed ?

**Comment by m-henderson** [comment]
Looks like the get a hint button needs to be directed to this url: https://forum.freecodecamp.org/t/freecodecamp-challenge-guide-accessing-nested-objects-in-json

**Comment by randelldawson** [comment]
The link to this challenge shows the message "Oops! That pge doesn't exist or is private".  It appears it need to be change to the following URL.
https://forum.freecodecamp.org/t/freecodecamp-challenge-guide-accessing-nested-objects-in-json/16161

**Comment by nsuchy** [comment]
@randelldawson I thought the plan was updating all links to the freeCodeCamp Guide once the challenge guide is complete. Perhaps we should put this on hold until then?

**Comment by randelldawson** [comment]
@nsuchy - I was just reporting what I found after a camper posted something about it when they were working on the challenge.  I referred them to the correct link shown above by @m-henderson.

**Comment by m-henderson** [comment]
I remember trying to do this in the past and I couldn't find where freecodecamp stores the Get a hint links.
